packages/MarkdownEditor-Tests.package/MarkdownEmphasisDelimiterTest.class/instance/testDifferentDelimitersCreateBoldAttribute.st

```diff
@@ -1,17 +1,11 @@
-tests - attributes
+tests - emphases
 testDifferentDelimitersCreateBoldAttribute
```

Comment:
`testDifferentDelimitersCreateStrongEmphasis`

Reply:
I know that `BoldEmphasis` matches `ItalicEmphasis`, but there is only `StrongEmphasis` in Markdown-talk.
How about `StrongEmphasis` and `NormalEmphasis` / `SimpleEmphasis`? (alternatives welcome)